analysis-configs/gradient_analysis_config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "version": "1.0.0",
+    "format_version": "1.0",
+    "input_path": "out/gradients_X/jitter/",
+    "output_path": "out/gradients_X/",
+    "ingredient_key" : "ext_A",
+    "run_similarity_analysis": true,
+    "save_plots": true,
+    "max_plots_to_save": 1
```

Comment:
TODO: formalize the analysis config options and add default settings for all of them